core/config/v2/env.go

```diff
@@ -0,0 +1,10 @@
+package v2
+
+import (
+	"os"
+	"strings"
+)
+
+var (
+	CLDev = strings.ToLower(os.Getenv("CL_DEV")) == "true"
```

Comment:
What is the purpose of this file? Is it a collection of attributes that we decided to keep as env vars? Will there be more?

Reply:
Yeah, there will be more, and they may need more complicated helpers for different types. At least secrets, maybe others https://app.shortcut.com/chainlinklabs/story/33624/add-secrets-toml